Source/ColorizeSwift.swift

```diff
@@ -59,6 +61,11 @@ public struct TerminalStyle {
 }
 
 extension String {
+
+    public static var enableColor = true
```

Comment:
The `enableColor` sounds like an action.With that naming convention it should be a function not the property. We want to represent the state. It should be noun not a verb. As it is a boolean it should also be prefixed with "is" e.g. `isColorizationEnabled`.